fastify.js

```diff
@@ -56,6 +56,7 @@ function fastify (options) {
 
   validateBodyLimitOption(options.bodyLimit)
 
+  const connectionTimeout = options.connectionTimeout || defaultInitOptions.connectionTimeout
```

Comment:
There is no need to assign this to a variable.
```suggestion
```

Reply:
Good point. Done.